webapp/src/main/resources/config/application.properties

```diff
@@ -28,6 +28,7 @@ validation.resourceUrl=http://localhost:8080/api/resources/
 services.update.interval=${SERVICES_UPDATE_INTERVAL:0 0 0/2 * * *}
 
 mocks.enable-invocation-stats=${ENABLE_INVOCATION_STATS:true}
+mocks.enable-invocation-logs=${ENABLE_INVOCATION_LOGS:true}
```

Comment:
It would be better to disable this by default and make it explicitly opted-in. At least during the first releases, to be sure we don't have regression issues.